pkg/sql/catalog/descs/descriptor.go

```diff
@@ -98,27 +100,59 @@ func (tc *Collection) AddUncommittedComment(
 	tc.uncommittedComments.upsert(descID, subID, cmtType, cmt)
 }
 
-// GetZoneConfig fetches zone config from uncommitted cache if it exists,
-// otherwise the cached storage version.
-func (tc *Collection) GetZoneConfig(descID descpb.ID) *zonepb.ZoneConfig {
-	zc := tc.GetZoneConfigWithRawByte(descID)
-	if zc != nil {
-		return zc.ZoneConfig()
+// GetZoneConfigWithRawByte is similar to GetZoneConfigsWithRawByte but only
+// fetches for one id.
+func (tc *Collection) GetZoneConfigWithRawByte(
```

Comment:
If like I suggest in another PR you define a `catalog.ZoneConfig` interface, then you should just call this `GetZoneConfig`

Reply:
Yes, changed as suggested :)